index.html

```diff
@@ -395,5 +395,40 @@ <h6>Skills</h6>
         </div>
       </section>
     </main>
+    <footer>
+      <div class="container-about-me">
+        <form action="https://formspree.io/f/xoqbwkba" method="post">
+          <ul class="form-list">
+            <li><h2>Contact me</h2></li>
+            <li>
+              <p>
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d
+                love to help with it
+              </p>
+            </li>
+            <li class="form-list-text">
+              <input type="text" id="username" name="username" placeholder="username" maxlength="30" />
+            </li>
+            <li class="form-list-text">
+              <input type="email" id="email" placeholder="email" name="email" />
+            </li>
+            <li class="form-list-text">
+              <textarea
+                name="text-description"
+                id=""
+                cols="30"
+                rows="10"
+                maxlength="50"
+                placeholder="write your message here"
+              ></textarea>
+            </li>
+            <li>
+              <button class="button">Get in touch</button>
```

Comment:
- [ ] Kindly make sure the role of your button is to submit the filled information. I highly recommend using the button `type: submit`so that you will be able to submit your form correctly. 😉